src/java.base/share/classes/javax/crypto/CryptoPermissions.java

```diff
@@ -43,17 +43,17 @@
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 /**
- * This class contains CryptoPermission objects, organized into
+ * This class contains <code>CryptoPermission</code> objects, organized into
```

Comment:
Use {@code } for all of the javadoc in this class?

Reply:
using `{@code }`